### DISCUSSION TITLE
How can I assign a command output to an environment variable?

### DISCUSSION THREAD

**Discussion by grindarius** [nushell/nushell#11988] [Q&A] [answered]
I was trying to get GPG signing to work on my MacOS with nushell and I have to assign this command

```bash
export GPG_TTY=$(tty)
```

How can I have the equivalent command inside `env.nu` file? Thank you.

**Comment by fdncred** [comment, marked as answer]
You should read our docs at nushell.sh.

```nushell
$env.GPG_TTY = (tty)
```
if you notice an extra line ending you can do
```nushell
$env.GPG_TTY = (tty | str trim)
```
